src/main/java/mate/jdbc/model/Driver.java

```diff
@@ -0,0 +1,76 @@
+package mate.jdbc.model;
+
+import java.util.Objects;
+
+public class Driver {
+    private Long id;
+    private String name;
+    private String licenseNumber;
+
+    public Driver() {
+
+    }
+
+    public Driver(String name, String licenseNumber) {
+        this.name = name;
+        this.licenseNumber = licenseNumber;
+    }
+
+    public Driver(Long id, String name, String licenseNumber) {
+        this.id = id;
+        this.name = name;
+        this.licenseNumber = licenseNumber;
```

Comment:
```suggestion
        this.id = id;
        this(name, licenseNumber)
```

we can call one constructor from another

Reply:
fixed